libs/orders/src/lib/components/order-list/order-list.spec.tsx

```diff
@@ -166,7 +166,7 @@ describe('OrderListTable', () => {
       expect(mockCancel).toHaveBeenCalledWith(order);
     });
 
-    it('doesnt show buttons for liquidity provision orders', async () => {
+    it('does not show buttons for liquidity provision orders', async () => {
```

Comment:
suggestion (non blocking)
```suggestion
    it('shows if an order is a liquidity provision order and does not show order actions', async () => {
```